### PR TITLE
Close windows opened in tests

### PR DESCRIPTION
**Pull request by timholy**
I was running the tests in the REPL, and wondered if they had failed.

### CODE REVIEW

File: test/windowclose.jl [hunk 1]
```diff
@@ -11,6 +11,9 @@ window = GLFW.CreateWindow(800, 600, "InexactError")
 ccall( (:glfwSetWindowShouldClose, GLFW.lib), Void, (GLFW.WindowHandle, Cint), window, 189)
 @test GLFW.WindowShouldClose(window)
 
+sleep(0.25)
```
Comment (jayschwa) [inline test/windowclose.jl]: Why sleep?

Comment (timholy) [inline test/windowclose.jl]: No reason. Just wondered if there were events that might need a little time to sync. I'll delete if you prefer.

Comment (jayschwa) [inline test/windowclose.jl]: Please remove. If it doesn't work without a sleep, that could be indicative of other problems.

Comment (timholy) [inline test/windowclose.jl]: Done. There's another `sleep` at the very end of the file; I left that one in place.

**Comment by jayschwa** [comment]
Thanks for PR 💯!

**Pull request merged**